homeassistant/components/hassio/handler.py

```diff
@@ -49,6 +49,19 @@ async def _wrapper(*argv, **kwargs):
     return _wrapper
 
 
+def api_valid(func):
```

Comment:
What's the difference between this decorator and the one above?

Reply:
And why would we need two ?

Reply:
We could reuse `_api_data` if we make it non private.

Reply:
Then let's do that.